django_prices/forms.py

```diff
@@ -22,42 +27,39 @@ def __init__(
         *args,
         **kwargs
     ):
-        self.currency = currency
+        fields = (forms.DecimalField(), forms.ChoiceField(choices=available_currencies))
```

Comment:
What about passing min/max, number of digits and decimal places?

Reply:
I've added passing `max_digits` and `decimal_places`; it was checked by `MoneyPrecisionValidator`, but on money validation. Passing it to `DecimalField` implies it will be checked earlier, and the validation messages are cleaner.

I've thought a lot about min/max values we had. I think when we use many currencies it doesn't have any sense to keep `max_value` and `min_value` as we had when we extended `forms.DecimalField`. For field with, e.g., USD and EUR as available currencies, I have no idea what could mean `max_value=400`.
Instead of that I introduced `max_values` and `min_values` arguments, which expect list of `Money` objects, and each of them will be added to the field validation.